x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/Querier.java

```diff
@@ -117,7 +117,7 @@ public void query(List<Attribute> output, QueryContainer query, String index, Ac
         }
 
         // set runtime mappings
-        if (this.cfg.runtimeMappings() != null && this.cfg.runtimeMappings().isEmpty() == false) {
+        if (this.cfg.runtimeMappings() != null) {
```

Comment:
Not insisting, it's a "safety net":  Since we initialise the runtimeMappings with an empty map, maybe we could also remove this check.